package.json

```diff
@@ -18,7 +18,8 @@
     "start": "node -r esm src/index.js",
     "lint": "eslint src --ext .js",
     "test": "jest",
-    "check-url": "node -r esm src/verifyUrl.js"
+    "check-url": "node -r esm src/verifyUrl.js",
+    "check-refs": "node -r esm src/checkRefs.js"
```

Comment:
I propose to add this script to the the [release job](https://github.com/SocialGouv/fiches-travail-data/blob/master/.github/workflows/release.yml#L33)

Reply:
like [this](https://github.com/SocialGouv/fiches-travail-data/pull/5/files#diff-56c30d9224897b08c28075052d4595c5R34) ?